### ISSUE TITLE
VERY IMPORTANT

### ISSUE THREAD

**Issue by Artriix** [Zrips/Jobs#1735]
**Detailed description of the issue:**
The jobs when they take money, for example when placing a mineral. Every time the payment is made to the player, that money is being taken away in a loop, even if you restart the server, it is solved by obtaining money that exceeds what they are taking from you.

---

**CONFIG:**
Default files

---

**Jobs version:** 
5.2.2.3

**Server Type (Spigot/Paper/etc):** 
Paper

**Server Version:** 
Paper version git-Paper-17 (MC: 1.20) (Implementing API version 1.20-R0.1-SNAPSHOT) (Git: c287e92)

**Relevant plugins (Delete if this isn't needed):**
Essentials Economy

**Comment by Zrips** [comment]
Could you elaborate more on this issue as its hard to understand what exactly is the issue and how to reproduce it. Are we talking about negative payments?

**Comment by Artriix** [comment]
Yes, I'm talking about negative payments. It's like the variable is not reset to 0 once the player is charged. You can see the error by doing the following:
1 install plugin
2 enter the miner job
3 place a diamond ore
4 wait to receive negative payments.

**Comment by Zrips** [comment]
Some blocks are set to give you negative amounts when you place them, this is mostly relevant when trying to place diamond ore blocks to prevent from players placing and then breaking same block without any consequences. You can disable this behavior by simply removing black place action from appropriate jobs with negative payments.
If this is not the issue, then you will need to be more clear. Are you constantly getting money deduction when you are not doing anything or is this only when you place diamond ore blocks yet you still gain payments correctly when breaking something like stone blocks?

**Comment by Artriix** [comment]
Yes, they are constantly taking it away from me with each player payment without having placed more blocks, you can try it by doing the steps I told you.

**Comment by Zrips** [comment]
I did tried it and it doesn't continue with payment deductions. Can you test this without any plugin, just Jobs and your economy plugin? In case there is another plugin which interferes with all of this can causing this issue

**Comment by Amirshahnavazi64** [comment]
- [ ] all

**Comment by Amirshahnavazi64** [comment]
50000000000000

**Comment by Amirshahnavazi64** [comment]
500000000000

**Comment by Amirshahnavazi64** [comment]
50000000000

**Comment by Amirshahnavazi64** [comment]
![Screenshot_2024-07-19-21-35-18-274_com tronlinkpro wallet](https://github.com/user-attachments/assets/26b4c7cb-01c5-40a1-9898-2fe290fcb88c)
